convene-web/app/models/invitation.rb

```diff
@@ -0,0 +1,2 @@
+class Invitation < ApplicationRecord
```

Comment:
We may want to add a `belongs_to :space, inverse_of: :invitations`. See [the general belongs_to guide](https://guides.rubyonrails.org/association_basics.html#belongs-to-association-reference) and the [inverse_of option](https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-inverse-of)